api/v1/flow/flow.proto

```diff
@@ -517,6 +517,8 @@ message FlowFilter {
 
     // only return Flows that were classified with a particular verdict.
     repeated Verdict verdict = 5;
+    // only applicable to Verdict = DROPPED.
+    repeated DropReason drop_reason_desc = 35;
```

Comment:
I'm not entirely sure whether the field name should be `drop_reason_desc` or `drop_reason` would be more appropriate here.

Reply:
I think this is fine.

Reply:
Why 35 though? As far as I can see the highest proto field number for FlowFilter before this patch was `http_header = 32`.

Reply:
` repeated string cel_expression = 33;` 

CEL has used 33, considering that CEL is an experimental feature just introduced, therefore 34 is reserved for the future expansion of CEL.

Reply:
`cel_expression` is part of the `Experimental` message and thus doesn't have to follow the `FlowFilter` protobuf field numbers, added a commit in [this PR](https://github.com/cilium/cilium/pull/32130) to fix both https://github.com/cilium/cilium/pull/32130/commits/6a8f572d47f9aa417fbab9fee2dd3b13aa0819d6.